main.nf

```diff
@@ -10,149 +10,2962 @@
 */
 
 def helpMessage() {
-    // TODO nf-core: Add to this help message with new command line parameters
     log.info nfcoreHeader()
     log.info"""
 
     Usage:
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/viralrecon --reads '*_R{1,2}.fastq.gz' -profile docker
+    nextflow run nf-core/viralrecon --input samplesheet.csv --genome 'NC_045512.2' -profile docker
```

Comment:
Do we actually need quotes around `NC_045512.2`?

Reply:
Not sure either

Reply:
So you dont on the command-line but you do in the config. Removing the quotes from this line will generate an error:
https://github.com/nf-core/viralrecon/blob/5ad5c00c81bb67c3eb32d87cf21c85b7d3c7e866/conf/test.config#L26

```nextflow
N E X T F L O W  ~  version 20.01.0
Launching `main.nf` [wise_poisson] - revision: 0e179236d9
Unable to parse config file: 'nfcore/viralrecon/nextflow.config'

  Compile failed for sources FixedSetSources[name='/groovy/script/ScriptF797F2B40A2F57598333B02FE285C19D']. Cause: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:
  /groovy/script/ScriptF797F2B40A2F57598333B02FE285C19D: 26: unexpected token: 2 @ line 26, column 22.
       genome = NC_045512.2
                          ^

  1 error
```

Maybe best to just leave it as it is for consistency? 🤷